src/main/java/com/flab/bbt/auth/service/AuthService.java

```diff
@@ -28,8 +28,13 @@ private boolean isDuplicatedEmail(String email) {
         return userRepository.findByEmail(email).isPresent();
     }
 
-    public User authenticate(User user) {
-        return userRepository.findByEmailAndPassword(user.getEmail(), user.getPassword())
-            .orElseThrow(() -> new CustomException(ErrorCode.USER_NOT_FOUND));
+    public Long authenticate(User user) {
+        Optional<User> foundUser = userRepository.findByEmailAndPassword(user.getEmail(), user.getPassword());
+
+        if (foundUser.isEmpty()) {
+            return -1L;
```

Comment:
(minor)
return 값을 -1로 주고, 호출한 argumentResolver에서는 0보다 작은값인지 체크를 하고 있습니다.

type이 `Long`이니 null을 넣을 수 있고 이를 통해서 전달하는게 더 명시적이지 않을까요?
-> 

```suggestion
            return null;
```

src/main/java/com/flab/bbt/auth/controller/AuthController.java
```
        if (userId == null) {
            return CommonResponse.fail(ErrorCode.USER_NOT_FOUND);
        } 
```

Reply:
생각해보니 -1 값은 클라이언트 입장에서 좋지 않은 응답값인 거 같습니다. 특히 authController에서 `userId < 0` 이 부분도 -1 외의 다른 값도 있을 수도 있나 싶을 거 같은데, 컨트롤러에서 authService.authenticate 를 통해 유저가 인증되었는지 여부만 알면 되고 즉 null 을 리턴해주기만 하면 충분할 거 같습니다. 수정해 두겠습니다.